declarations/spell.d.ts

```diff
@@ -0,0 +1,32 @@
+/** @noSelfInFile */
+
+declare const BOOKTYPE_SPELL = 'spell';
+declare const BOOKTYPE_PET = 'pet';
+
+declare type WowBookType = 'spell' | 'pet';
```

Comment:
This is how i have done this in the past. This leads to better handle renaming for specific types.

```suggestion
declare type WowBookType = BOOKTYPE_SPELL | BOOKTYPE_PET;
```

Reply:
Done. had to use typeof, because these are also global constants, not just types.